Shared/Samples/Augment reality to collect data/AugmentRealityToCollectDataView.swift

```diff
@@ -0,0 +1,210 @@
+        graphicsOverlay.sceneProperties.surfacePlacement = .absolute
+        return graphicsOverlay
+    }()
+    /// The status text displayed to the user.
+    @State private var statusText = "Tap to create a feature"
+    /// The selected tree health for the new feature.
+    @State private var treeHealth: TreeHealth?
+    /// A Boolean value indicating whether a feature can be added .
+    @State private var canAddFeature = false
+    /// A Boolean value indicating whether the tree health action sheet is presented.
+    @State private var treeHealthSheetIsPresented = false
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    init() {
+        let featureLayer = FeatureLayer(featureTable: featureTable)
+        featureLayer.sceneProperties.surfacePlacement = .absolute
+        scene.addOperationalLayer(featureLayer)
+    }
+    
+    var body: some View {
+        if #available(iOS 16, *) {
+            NavigationStack {
+                augmentRealityToCollectDataView
+            }
+        } else {
+            NavigationView {
+                augmentRealityToCollectDataView
+            }
+        }
+    }
+    
+    @MainActor @ViewBuilder var augmentRealityToCollectDataView: some View {
+        VStack(spacing: 0) {
+            WorldScaleSceneView { _ in
+                SceneView(scene: scene, graphicsOverlays: [graphicsOverlay])
+            }
+            .calibrationButtonAlignment(.bottomLeading)
+            .onCalibratingChanged { isPresented in
+                scene.baseSurface.opacity = isPresented ? 0.5 : 0
+            }
+            .onSingleTapGesture { _, scenePoint in
+                graphicsOverlay.removeAllGraphics()
+                canAddFeature = true
+                
+                // Add feature graphic.
+                graphicsOverlay.addGraphic(Graphic(geometry: scenePoint))
+                statusText = "Placed relative to ARKit plane"
+            }
+            .task {
+                do {
+                    try await featureTable.load()
+                } catch {
+                    self.error = error
+                }
+            }
+            .overlay(alignment: .top) {
+                Text(statusText)
+                    .multilineTextAlignment(.center)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(8)
+                    .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
+            Divider()
+        }
+        .toolbar {
+            ToolbarItemGroup(placement: .bottomBar) {
+                Button {
+                    treeHealthSheetIsPresented = true
+                } label: {
+                    Image(systemName: "plus")
+                        .imageScale(.large)
+                }
+                .disabled(!canAddFeature)
+                .actionSheet(isPresented: $treeHealthSheetIsPresented) {
```

Comment:
This modifier is deprecated:

![image](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/2257493/066d2959-cf81-49f2-aa0e-fd4a55d756e8)

Please use a replacement.

Reply:
I used `confirmationDialog` but the `titleKey` does not show when set to `"Add Tree"` or `Text("Add Tree")`. Do you have any ideas on how to fix this?

Reply:
Nevermind, setting the `confirmationDialog.titleVisibility` to `visible` resolves the issue.